core/src/HIP/Kokkos_HIP_Parallel_Team.hpp

```diff
@@ -701,9 +689,7 @@ class ParallelReduce<FunctorType, Kokkos::TeamPolicy<Properties...>,
     int64_t threadid = 0;
     if (m_scratch_size[1] > 0) {
       __shared__ int64_t base_thread_id;
-      // FIXME_HIP This uses g_device_hip_lock_arrays which is not working
       if (hipThreadIdx_x == 0 && hipThreadIdx_y == 0) {
-        Impl::hip_abort("Error should not be here (not implemented yet)\n");
```

Comment:
Why did you remove this code? `g_device_hip_lock_arrays` is not implemented in this PR.

Reply:
Restored in latest.